templates/layout.html

```diff
@@ -1,12 +1,13 @@
 {%- from "macros/translation.html" import transbag -%}
 
 {%- macro menu_item(identifier) -%}
-  {%- set identifier_url = '/' + identifier -%}
-  {%- set trans_url = identifier_url|url(alt=this.alt) -%}
-  {%- set this_is_child = this.is_child_of(identifier_url) -%}
-  <li class="nav-item{% if this_is_child %} active{% endif %}">
-    <a class="nav-link" href="{{ trans_url }}">{{- transbag('menu', this.alt, identifier) -}}{%- if this_is_child -%}<span class="sr-only">(current)</span>{%- endif -%}</a>
-  </li>
+{%- set identifier_url = '/' + identifier -%}
+{%- set trans_url = identifier_url|url(alt=this.alt) -%}
+{%- set this_is_child = this.is_child_of(identifier_url) -%}
+<li class="nav-item{% if this_is_child %} active{% endif %}">
+  <a class="nav-link" href="{{ trans_url }}">{{- transbag('menu', this.alt, identifier) -}}{%- if this_is_child -%}<span
+      class="sr-only">(current)</span>{%- endif -%}</a>
+</li>
```

Comment:
Um... you haven't "fixed the indentation".

Take this block for example. It de-indents 8 lines of code, and re-flows one long line. AFAICT, there is *no* substantive change that alters the behavior of the menu. But it shows up as an 8 line diff.

This is just the first diff in the patch. By my count, you have to get down to the 6th delta before you get to something that could be called substantive. 

The PR should only contain the changes that are *necessary* to fix the problem. A little light cleaning up is sometimes acceptable, when it doesn't blur the purpose of the patch. 

In this case, it's almost impossible to tell what is an important change, and what is "cleaning up".

Reply:
My All indentation and Changed are fixed please check it.. and i request to merge it

Reply:
No, you have not. Please read what I wrote.

Reply:
I don't understand what you want!

Reply:
You have presented a PR. That PR is a diff against the existing codebase. There are changes in that diff that *do not* relate to the stated problem - fixing the layout of the menu on narrow displays.

As an example, take the very first line in the PR that has changed:
`
{%- set identifier_url = '/' + identifier -%}
`
You've removed 2 spaces from the start of that line.  The content of the line is otherwise identical.

In this *specific* case, it's not too bad, because the line is still aligned in the diff; but a few lines later, you've de-indented the `<head>` block for some reason... and as a result, Github is no longer able to work out which pieces of the diff line up with each other.

I don't just look at the result of the PR - I try to understand the change you've made, and work out if you've implemented the change the right way. In it's current state, I can't do that easily, because you've mixed important changes (altering the rendering of the menu) with cosmetic changes (altering indentation).

You need to revert the changes in this PR that are related to code formatting. Don't change the existing indentation unless the change in indentation is *required* to fix the bug you're trying to fix.

Reply:
Dear Sir,
I think now fixed indentation and because of I am student and I doing this job and if you merge this pull request then I am very happy so please merge it!

I used VSCode so for the html I used format document feature so that feature, I think not justify jinja templates indentation, but I can do it manually.
Please review my PR request

Thankyou.
Gagan Aggarwal
gaggarwal124@gmail.com